modules/fundamental/src/endpoints.rs

```diff
@@ -8,19 +10,20 @@ pub fn configure(
     config: &mut web::ServiceConfig,
     db: Database,
     storage: impl Into<DispatchBackend>,
+    auth: Option<Arc<Authenticator>>,
 ) {
     let storage = storage.into();
 
     let ingestor_service = IngestorService::new(Graph::new(db.clone()), storage.clone());
     config.app_data(web::Data::new(ingestor_service));
```

Comment:
Can't we inject auth into the config before we call all the configure's, obviating the need to pass it?

Reply:
I don't know how. I am open to suggestions.